smoke_test/helpers/http_status_validation.rb

```diff
@@ -0,0 +1,12 @@
+module HttpStatusValidation
+module_function
+
+  def validate_response_status!
+    if page.status_code.to_s.match(/^(4|5)\d\d/)
```

Comment:
`(400..599).include?(page.status_code)` would probably be most expressive, though `>= 400` would also work.